Makefile

```diff
@@ -318,6 +318,11 @@ GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
 GOLANGCI_LINT_VERSION ?= v1.51.1
 
 .PHONY: lint
-lint:
+lint: lint-metrics
 	test -s $(GOLANGCI_LINT) || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(LOCALBIN) $(GOLANGCI_LINT_VERSION)
 	$(GOLANGCI_LINT) run --timeout 5m
+
+
+.PHONY: lint-metrics
+lint-metrics:
+	./hack/prom_metric_linter.sh --operator-name="kubevirt" --sub-operator-name="ssp"
```

Comment:
I think `prom_metric_linter.sh` wasn't copied during image building. That's why it says in [1]:
```
bash: line 1: ./hack/prom_metric_linter.sh: No such file or directory
```
So try to add to SSP's dockerfile:
```
COPY hack/prom_metric_linter.sh hack/prom_metric_linter.sh
```

[1] https://storage.googleapis.com/origin-ci-test/pr-logs/pull/kubevirt_ssp-operator/621/pull-ci-kubevirt-ssp-operator-main-images/1677968867898429440/build-log.txt

Reply:
@0xFelix WDYT?

Reply:
I don't see that error anymore, after we've added `COPY` to Dockerfile.

Reply:
Good catch!